decidim-accountability/app/cells/decidim/accountability/project/show.erb

```diff
@@ -1,4 +1,4 @@
-  <h2 class="h2 decorator"><%= title %></h2>
+  <h1 class="h2 decorator" data-heading-tag><%= title %></h1>
```

Comment:
If this PR is about meetings, please split changes to other modules to other PRs.

Extending these PRs with unrelated changes makes reviewing them really heavy and confusing. The smaller the PR, the better.

Reply:
I disagree, _accountability_ is in develop right now. When a PR introduces changes like this one, we must update the develop code to make it work with the introduced change.

In any case, from now on this sort of issues won't come up again, since such changes (let's say, _components_ changes) will be isolated in specific PRs, in order to avoid messing around.